docs/configuration.rst

```diff
@@ -31,10 +31,10 @@ the secret key is leaked, it is imperative that a new secret key is created.
 
 Sessions
 --------
-Server-side session data is currently stored in Redis, you must therefore
-configure where your Redis instance is located using:
+Server-side session data can be saved in different data stores (e.g. Redis),
+you must therefore configure a factory that returns the store object:
```

Comment:
It would be more accurate to 'provide a factory '.
I believe that defining what kind of object we are expecting would be more helpfull 'returns the KV session store object', I'm nitpicking since we also mention it in the configuration.